impala/assets/dashboards/impala_overview.json

```diff
@@ -0,0 +1,1572 @@
+{
+  "title": "Impala - Overview",
+  "description": "## Impala\n\nThis dashboard provides a high-level overview of your Impala processes so you can monitor their performance and health.\n\n- [Official Impala integration docs](https://docs.datadoghq.com/integrations/impala/)\n\nClone this template dashboard to make changes and add your own graph widgets.",
+  "widgets": [
+    {
+      "id": 1171210667439246,
+      "definition": {
+        "title": "About Impala",
+        "title_align": "center",
+        "type": "group",
+        "banner_img": "/static/images/integration_dashboard/impala_hero_1.jpeg",
+        "show_title": false,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 7840582060409836,
+            "definition": {
+              "type": "note",
+              "content": "This is a high-level overview of your Impala processes, so you can track health status, search and indexing performance, and resource utilization metrics from all your services and be better prepared to address potential issues.\n\n##### [**Datadog Impala integration docs&nbsp;↗**](https://docs.datadoghq.com/integrations/impala/)",
+              "background_color": "transparent",
+              "font_size": "18",
+              "text_align": "left",
+              "show_tick": false,
+              "tick_pos": "50%",
+              "tick_edge": "left",
+              "has_padding": true
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 6,
+              "height": 2
+            }
+          }
+        ]
+      },
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 6,
+        "height": 5
+      }
+    },
+    {
+      "id": 11147134331060,
+      "definition": {
+        "title": "Global overview",
+        "title_align": "left",
+        "type": "group",
+        "background_color": "blue",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
```

Comment:
It's unclear at a glance what the numbers in this overview mean. I've also seen them turn red without changing the number, which is confusing. Can we make this clearer?

Reply:
This is the widget used to display the status of the service check. If it's all green with a number, it means that the three services are up, if they're red it means they're down. You can also have a mix of red and green.

You saw them red when I restarted my services locally. I can try to improve that but that's the default behaviour of the widget